tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -562,6 +562,14 @@ clean:
 clean_downloads:
 	rm -rf $(MAKEFILE_DIR)/downloads
 
+# Print internal variables (used by CMSIS-Pack build)
+printvar-%: 
```

Comment:
This is certainly the direction that we want to go on.

My only suggestion is to make the names explicit so that we have a clear notion of what needs to be supported moving forward.

I want to still be able to change the variable names in the Makefile without breaking users.

I'd say the following:
```make
print-tensorflow-lite-micro-cc-sources:
    @echo $(MICROLITE_CC_SRCS)

print-tensorflow-lite-micro-cc-headers:
    @echo $(MICROLITE_CC_HDRS)
```

Reply:
Created  https://github.com/MatthiasHertel80/tensorflow/pull/1 with my suggestion. If you are ok with that, then let's merge my PR into your branch and then merge your PR into tensorflow.